### DISCUSSION TITLE
Backup restore tier storage data

### DISCUSSION THREAD

**Discussion by gfes980615** [Altinity/clickhouse-backup#825] [Q&A]
Hi, team
 
I had used the [clickhouse-backup](https://github.com/Altinity/clickhouse-backup) to backup the clickhouse data
Especially, I used the AWS S3 to storage our tier storage (inner 7 days on disk - Hot, previous 7 days transfer to s3 - Cold)

When I use this tool I encounter some problems
I restore the latest backup name with the backup `watch` command, I got this error:
```
{"command":"restore --data shardshard0-increment-20240206003348","status":"error","start":"2024-02-06 01:57:22","finish":"2024-02-06 01:57:36","error":"can't attach data parts for table 'ck_job.otel_trace_lab': code: 233, message: Detached part \"20240205_0_20505_203\" not found"}
```
Which seems that only restore Hot data.
 
1. Does the clickhouse-backup can restore the tier storage data?
2. Or the tier storage I just need to restore the schema and the metadata, and then I can search the data in the tier storage?
 
------
The related current used service information as below:
clickhouse-operator: 0.22.2
clickhouse-server image: 23.8.9
clickhouse-backup: 2.4.17

**Comment by Slach** [comment]
could you share 
`clickhouse-backup print-config`?

logs for `create` / `upload` or `create_remote` command execution?

**Comment by gfes980615** [comment]
Hi, thanks for your reply!
We already fixed this issue by setting this config `send_metadata` refer to this [doc](https://kb.altinity.com/altinity-kb-setup-and-maintenance/altinity-kb-s3-object-storage/s3disk/)
Close the discussion
